src/Goteo/Payment/Method/PaypalPaymentMethod.php

```diff
@@ -28,8 +28,12 @@ public function purchase(): ResponseInterface
         /** @var ExpressGateway */
         $gateway = $this->getGateway();
         $invest = $this->getInvest();
-        $project = Project::get($invest->project);
-        $transactionId = sprintf("%s-%s", $project->getNumericId(), $invest->id);
```

Comment:
Fixed this to pad the transaction id with leading "0". This is to avoid confusion with past invests (pre project numeric id) that did go to a project but did not include the project numeric id in the reference number, this way we can still distinguish non-project invests while keeping records sorted.